.github/generate-envs.py

```diff
@@ -124,7 +124,7 @@
         "lang": "verilog",
         "sim": "icarus",
         "sim-version": "homebrew-stable",
-        "os": "macos-latest",
+        "os": "macos-11",
```

Comment:
These two are equivalent as of today, but I'd like to get our CI a bit more predictable by removing as many "variable versions" as possible.